db/factories/Shifts/ShiftFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
I would say all new files should `declare(strict_types=1);`.

Reply:
Then we should create a check that enforces it ;)